cpuinfo/cpuinfo.py

```diff
@@ -670,25 +696,14 @@ def _run_asm(self, *byte_code):
 	# FIXME: We should not have to use different instructions to
 	# set eax to 0 or 1, on 32bit and 64bit machines.
 	def _zero_eax(self):
-		if DataSource.bits == '64bit':
-			return (
-				b"\x66\xB8\x00\x00" # mov eax,0x0"
```

Comment:
The '\x66' prefix was causing only the lower 16-bits of eax to be cleared.  This caused bad output from cpuid